gladier_xpcs/tools/plot.py

```diff
@@ -6,18 +6,10 @@
 def make_corr_plots(**data):
     import os
     import json
-    import traceback
     from xpcs_webplot.plot_images import hdf2web_safe
     from xpcs_webplot import __version__ as webplot_version
-    
-    img_dir = os.path.join(data['proc_dir'], os.path.dirname(data['hdf_file']))
-    os.chdir(img_dir)
-    try:
-        h5filename = os.path.join(data['proc_dir'], data['hdf_file'])
-        hdf2web_safe(h5filename, target_dir=data['proc_dir'],
-                     image_only=True)
-    except (Exception, SystemExit):
-        return traceback.format_exc()
+
+    hdf2web_safe(data['hdf_file'], target_dir=data['proc_dir'], image_only=True)
```

Comment:
I can make target_dir absolute so that it won't create a new directory inside target_dir.